libs/openFrameworks/types/ofBaseTypes.h

```diff
@@ -878,17 +1714,20 @@ class ofBaseRenderer{
 	/// \param depth The depth of the box.
 	virtual void drawBox(const ofPoint& position, float width, float height, float depth) const;
 
-	/// \brief Draws a cube with the specified size, starting from the specified position.
+	/// \brief Draws a cube with the specified size, starting from the specified
+	/// position.
 	///
-	/// A cube is drawn starting from a 3D reference position, with the specified size.
-	/// The cube is drawn with the current color, e.g. set with ofSetColor().
-	/// The cube is drawn filled by default; change this with ofFill();
+	/// A cube is drawn starting from a 3D reference position, with the
```

Comment:
This description was already in the inline documentation. It may belong more in markdown rather than inline.